flepimop/gempyor_pkg/src/gempyor/statistics.py

```diff
@@ -238,22 +242,40 @@ def llik(self, model_data: xr.DataArray, gt_data: xr.DataArray) -> xr.DataArray:
             The log-likelihood of observing `gt_data` from the model `model_data` as an
             xarray DataArray with a "subpop" dimension.
         """
+
         dist_map = {
-            "pois": scipy.stats.poisson.logpmf,
+            "pois": lambda ymodel, ydata: -(ymodel + 1)
+            + ydata * np.log(ymodel + 1)
+            - gammaln(ydata + 1),
```

Comment:
While I don't mind this version of the poisson PMF (but prefer the former impleementation based on Scipy.stat), adding one the model ouputs is not documented and it should instead use the zero_to_one config options ((though first zero to one just change zero to one instead of adding one to every number, and does it for both ydata and ymodel and here it's done just for model)).

Just for reference  Poisson log-like hood with this formula is `log(L) = ydata * log(ymodel) - ymodel - log(factorial(ydata))`
and here we have
`ydata * np.log(ymodel + 1) -(ymodel + 1)  - gammaln(ydata + 1)`


(and note that `gammaln(ydata + 1) ==  log(factorial(ydata))` because gammaln(n + 1) == log(n!), so that bit is correct)

Reply:
See GH-420

Reply:
Thanks for solving this!